HGustavs/LenaSYS
# Added ability to choose between candidate and primary keys #12158

Related to main-issue #11744 
and sub-issue #12158 

The user can now edit the type of keys on attributes. Note, the previously type namned 'key' is now split into two categories and are called 'candidate' and 'primary'. If an attribute is primary, only the primary keys will have underlines in the ER-table. If no primary keys are present all candidate keys are underlined instead. 

Also note that multiple primary keys and / or candidate keys can exist on the same attribute, this it to allow the user to learn by misstakes and not stop possible misstakes.

To test:

- Try adding more primary keys to an entity. Check if only primary keys are underlined in ER-table.
- Try adding more candidate keys to an entity and one or more primary keys. Check that only the primary key is underlined in the ER-table.
- Try changing the primary to a candidate. Check that all candidate are now underlined in the ER-table.
